x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/rules/logical/ReplaceRegexMatchTests.java

```diff
@@ -60,19 +45,6 @@ public void testMatchAllRLikeToExist() {
         assertEquals(fa, inn.field());
     }
 
-    public void testExactMatchLike() {
-        for (String s : asList("ab", "ab0%", "ab0_c")) {
```

Comment:
Hm, this is not a _perfect_ duplicate of the test below; `testExactMatchWildcardLike` does not test escaping like this one does.

Should we expand `testExactMatchWildcardLike` to include the (actually exact) string patterns `ab\*` and `ab\?c`?